README.md

```diff
@@ -39,36 +39,38 @@ import React, { useState, useContext } from "react";
 import createContainer from "constate";
 
 // 1️⃣ Create a custom hook as usual
-function useCounter() {
+const useCounter = () => {
   const [count, setCount] = useState(0);
   const increment = () => setCount(prevCount => prevCount + 1);
   return { count, increment };
 }
 
-// 2️⃣ Create container
-const CounterContainer = createContainer(useCounter);
+// 2️⃣ When you need to share your state, simply wrap your hook
+//    with the createContainer higher-order hook, like so:
+const useCounter = createContainer(() => {
+   // ... same logic here
+});
 
 function Button() {
-  // 3️⃣ Use container context instead of custom hook
-  // const { increment } = useCounter();
-  const { increment } = useContext(CounterContainer.Context);
+  // 3️⃣ Use the same hook like you were before, with the same api. No change.
+  const { increment } = useCounter()
```

Comment:
The hook returned by `createContainer` isn't 100% equivalent to the wrapped hook. You can't pass arguments to the hook, for example.